server/graphql/v1/mutations/collectives.js

```diff
@@ -380,6 +380,37 @@ export function editCollective(_, args, req) {
 
   let collective, parentCollective;
 
+  const tiers = args.collective.tiers;
+  forEach(tiers, tier => {
+    const presets = tier.presets || [];
+    const amount = tier.amount;
+    const name = tier.name;
+    const amountType = tier.amountType;
```

Comment:
What if `amountType` is not sent by the client?

Should we make it mandatory to deploy frontend first or can we have a simple transition strategy (better)?

Reply:
The backend should be deployed first, since we can easily determine `amountType`  from the tier data without being sent by the frontend. 
Basically I just need to add:
```js
tier.amountType = tier.presets? 'FLEXIBLE' : 'FIXED'
```

Reply:
Ok, let's do that.